website/src/content/development/notifications.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Notifications
+---
+
+In the previous sections, we learned how to develop forms. In this section we look at how to communicate feedback to users using notifications, in particular in response to user actions.
+
+To facilitate the use of notifications, Custom Applications come with a built-in notification system.
+
+# Notification system
+
+Notifications are an important way of providing feedback in response to user actions.
+For example submitting a form might succeed or fail and as a result the user should be notified about that in a way that is understandable and easy to find.
+
+Notifications are grouped by what we call "domains". A domain indicates where the notification should be displayed on the page.
```

Comment:
Below we are listing the different types so maybe we could do the same here:
```suggestion
Notifications are grouped by what we call "domains". A domain indicates where the notification should be displayed on the page (`global`, `page` and `side`).
```

Reply:
Thanks, I added `page` an `side` (and omitted `global` since we don't want to "advocate" for using that)

Reply:
Ok. Good to know 😄